turbo.json

```diff
@@ -1,30 +1,148 @@
 {
   "$schema": "https://turbo.build/schema.json",
   "tasks": {
-    "build": {
+    // @modrinth/app
+    "@modrinth/app#build": {
       "dependsOn": ["^build"],
-      "inputs": ["$TURBO_DEFAULT$", ".env*"],
-      "outputs": [".nuxt/**", "dist/**", ".output/**", "target/**"],
-      "env": [
-        "NODE_ENV",
-        "SITE_URL",
-        "BASE_URL",
-        "FLAG_OVERRIDES",
-        "BROWSER_BASE_URL",
-        "RATE_LIMIT_IGNORE_KEY",
-        "VERCEL_*",
-        "CF_PAGES_*",
-        "HEROKU_APP_NAME"
-      ]
+      "inputs": ["$TURBO_DEFAULT$", "!**/*.md", "!LICENSE"],
+      "outputs": ["target"]
     },
-    "lint": {},
-    "dev": {
+    "@modrinth/app#dev": {
```

Comment:
You removed `"env": [ "*" ]` for dev, which means I get `cannot open display:`. You have to add `DISPLAY` and some more flags.

Reply:
`WEBKIT_DISABLE_DMABUF_RENDERER`